bin/run-couch-master-on-travis.sh

```diff
@@ -4,12 +4,6 @@ set -x
 
 CWD=$(pwd)
 
-# Rebar isnt in apt
-git clone git://github.com/rebar/rebar.git ~/rebar
-cd ~/rebar
-./bootstrap
-export PATH=$(pwd):$PATH
-
 # Sweet, build CouchDB
 cd ..
```

Comment:
This `cd  ..` should get also removed since it's about get out from rebar directory.